test/acceptance/environments/local.yml

```diff
@@ -101,3 +101,13 @@ httpMockUrl: http://localhost:9000
 elasticsearch:
   addresses:
     - http://localhost:9200
+
+# s3
+s3-autoclean: true
+bucket: golium-demo
```

Comment:
I will prefix it with s3, as you do with minio ones.

Reply:
Done